linetools/spectra/xspectrum1d.py

```diff
@@ -1030,7 +1039,7 @@ def fit_continuum(self, knots=None, edges=None, wlim=None, dw=10.,
             [tuple(pts) for pts in wrapper.contpoints])
         self.meta['contpoints'].sort()
 
-    def _interp_continuum(self, x, y):
+    def _interp_continuum(self, x, y, wv):
```

Comment:
Can you make `wv` a keyword argument with default value `None`, meaning that the spectrum wavelength is used? It seems fine to have a wavelength argument, but I think it makes sense to use the spectrum wavelength array of the spectrum by default.